cmd/minishift/cmd/start.go

```diff
@@ -108,7 +109,12 @@ For the latter see 'minishift config -h'.`,
 		Run: runStart,
 	}
 
+	// enable experimental flags
+	//startCmd.Flags().BoolVar(&future, "future", false, "Enable experimental features (unsupported)")
```

Comment:
```diff
diff --git a/cmd/minishift/cmd/root.go b/cmd/minishift/cmd/root.go
index 9662a31..12bdb90 100644
--- a/cmd/minishift/cmd/root.go
+++ b/cmd/minishift/cmd/root.go
@@ -160,6 +160,8 @@ func setFlagsUsingViper() {
 }
 
 func init() {
+       RootCmd.Flags().BoolVar(&future, "future", false, "Enable experimental features (unsupported)")
+
        RootCmd.PersistentFlags().Bool(showLibmachineLogs, false, "Show logs from libmachine.")
        RootCmd.AddCommand(configCmd.ConfigCmd)
        RootCmd.AddCommand(cmdOpenshift.OpenShiftCmd)
```

will result in `Unknown flag: --future` when called as `minishift --future` or `minishift start --future`... This was commented in https://github.com/minishift/minishift/issues/1170#issuecomment-318060388